src/io/flutter/run/bazel/BazelRunConfig.java

```diff
@@ -66,9 +68,9 @@ public LaunchState getState(@NotNull Executor executor, @NotNull ExecutionEnviro
       throw new ExecutionException(e);
     }
 
-    final MainFile main = MainFile.verify(launchFields.getEntryFile(), env.getProject()).get();
+    final Workspace workspace = fields.getWorkspace(getProject());
     final RunMode mode = RunMode.fromEnv(env);
-    final Module module = ModuleUtil.findModuleForFile(main.getFile(), env.getProject());
+    final Module module = ModuleUtil.findModuleForFile(workspace.getRoot(), env.getProject());
```

Comment:
Create a local reference to `workspace.getRoot()` as it is used 3 times in this body.

Reply:
Done.